spec/support/factories/band.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :band, class: "WasteExemptionsEngine::Band" do
+    name { "Band #{Faker::Lorem.word}" }
+    sequence(:sequence) { Faker::Number.between(from: 1, to: 10) } # reserved word
```

Comment:
Is this guaranteed to use unique values or do you need to add the unique keyword?

Reply:
Good idea. Thanks. Added unique